### PR TITLE
open-mpi.org

### LINKED ISSUE #349: +open-mpi (245/548)
Formulae: https://formulae.brew.sh/formula/open-mpi

### PR DESCRIPTION
**Pull request by and-ri**
closes #349

### CODE REVIEW

File: projects/open-mpi.org/package.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    - ./configure $ARGS
+    - make --jobs {{ hw.concurrency }} all
+    - make --jobs {{ hw.concurrency }} check
+    - make --jobs {{ hw.concurrency }} install
+    - install {{prefix}}/lib/*.mod {{prefix}}/include/
+    - sed -i.bak 's|{{prefix}}|${pcfiledir}/../..|g' {{prefix}}/lib/pkgconfig/*.pc
+    
+  env:
+    darwin:
+      MACOSX_DEPLOYMENT_TARGET: "$(sw_vers -productVersion)"
```
Comment (jhheider) [inline projects/open-mpi.org/package.yml]: This should be automatically 10.6, so we support versions we intend (i.e. you shouldn't have to set it). does it not build unless set?

File: projects/open-mpi.org/package.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  strip-components: 1
+
+versions:
+  github: open-mpi/ompi/tags
+
+dependencies:
+  open-mpi.org/hwloc: '*'
+  libevent.org: '*'
+  linux:
+    gnu.org/gcc: '*'
```
Comment (jhheider) [inline projects/open-mpi.org/package.yml]: Is this really a run-time and not a build-time dep? And doesn't darwin need a compiler as well?

Comment (and-ri) [reply projects/open-mpi.org/package.yml]: image
the homebrew formula states that gcc is needed for gfortran
hmm i must have excluded it for darwin during one of my attempts to assemble it and forgot to return it 😄

Comment (jhheider) [reply projects/open-mpi.org/package.yml]: fun. weird that it'd be runtime.

Comment (and-ri) [reply projects/open-mpi.org/package.yml]: If I return gcc for darwin I get:
```
configure:6591: checking whether the C compiler works
configure:6613: gcc -w   -Wl,-rpath,/Users/andrejrabcenko/.tea conftest.c  >&5
ld: library not found for -lSystem
collect2: error: ld returned 1 exit status
```

**Comment by jhheider** [comment]
well, that was painful.

**Pull request merged**